Marlin/src/feature/bedlevel/ubl/ubl_motion.cpp

```diff
@@ -39,7 +39,7 @@
   extern float destination[XYZE];
 
   #if AVR_AT90USB1286_FAMILY  // Teensyduino & Printrboard IDE extensions have compile errors without this
-    inline void set_current_to_destination() { COPY(current_position, destination); }
+    inline void set_current_to_destination() { COPY( destination,current_position); }
```

Comment:
This inverted process , make impossible to use M600 because of the different ways to use RUNPLAN and Doblockmove.
Now it works perfect

Reply:
If that works you have to use `set_destination_to_current()` everywhere in this file.
Dont mess with the function definitions,